plugins/aws/resoto_plugin_aws/resource/dynamodb.py

```diff
@@ -0,0 +1,363 @@
+        "dynamodb_replicas": S("Replicas", default=[]) >> ForallBend(AwsDynamoDbReplicaDescription.mapping),
+        "dynamodb_restore_summary": S("RestoreSummary") >> Bend(AwsDynamoDbRestoreSummary.mapping),
+        "dynamodb_sse_description": S("SSEDescription") >> Bend(AwsDynamoDbSSEDescription.mapping),
+        "dynamodb_archival_summary": S("ArchivalSummary") >> Bend(AwsDynamoDbArchivalSummary.mapping),
+        "dynamodb_table_class_summary": S("TableClassSummary") >> Bend(AwsDynamoDbTableClassSummary.mapping),
+    }
+    arn: Optional[str] = field(default=None)
+    dynamodb_attribute_definitions: List[AwsDynamoDbAttributeDefinition] = field(factory=list)
+    dynamodb_key_schema: List[AwsDynamoDbKeySchemaElement] = field(factory=list)
+    dynamodb_table_status: Optional[str] = field(default=None)
+    dynamodb_provisioned_throughput: Optional[AwsDynamoDbProvisionedThroughputDescription] = field(default=None)
+    dynamodb_table_size_bytes: Optional[int] = field(default=None)
+    dynamodb_item_count: Optional[int] = field(default=None)
+    dynamodb_billing_mode_summary: Optional[AwsDynamoDbBillingModeSummary] = field(default=None)
+    dynamodb_local_secondary_indexes: List[AwsDynamoDbLocalSecondaryIndexDescription] = field(factory=list)
+    dynamodb_global_secondary_indexes: List[AwsDynamoDbGlobalSecondaryIndexDescription] = field(factory=list)
+    dynamodb_stream_specification: Optional[AwsDynamoDbStreamSpecification] = field(default=None)
+    dynamodb_latest_stream_label: Optional[str] = field(default=None)
+    dynamodb_latest_stream_arn: Optional[str] = field(default=None)
+    dynamodb_global_table_version: Optional[str] = field(default=None)
+    dynamodb_replicas: List[AwsDynamoDbReplicaDescription] = field(factory=list)
+    dynamodb_restore_summary: Optional[AwsDynamoDbRestoreSummary] = field(default=None)
+    dynamodb_sse_description: Optional[AwsDynamoDbSSEDescription] = field(default=None)
+    dynamodb_archival_summary: Optional[AwsDynamoDbArchivalSummary] = field(default=None)
+    dynamodb_table_class_summary: Optional[AwsDynamoDbTableClassSummary] = field(default=None)
+
+    @classmethod
+    def collect(cls: Type[AwsResource], json: List[Json], builder: GraphBuilder) -> None:
+        def add_instance(table: str) -> None:
+            table_description = builder.client.call("dynamodb", "describe-table", "Table", TableName=table)
+            instance = cls.from_api(table_description)
+            builder.add_node(instance, table_description)
+            builder.submit_work(add_tags, instance)
+
+        def add_tags(table: AwsDynamoDbTable) -> None:
+            tags = builder.client.list("dynamodb", "list-tags-of-resource", "Tags", ResourceArn=table.arn)
+            if tags:
+                table.tags = bend(ToDict(), tags)
+
+        for table in json:
+            builder.submit_work(add_instance, table)
+
+    def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
```

Comment:
If you have a `connect_in_graph` method, you should always define a `reference_kinds` variable, which documents the kind of connection that is made.